rust-app/Cargo.toml

```diff
@@ -30,14 +30,15 @@ speculos = [
     "ledger-parser-combinators/logging",
 ]
 extra_debug = ["ledger-log/log_trace"]
+pending_review_screen = []
```

Comment:
So this has always been missing?

Reply:
Seems so, but now there was a warning that this was missing